Content.Shared/Movement/Components/MovementSpeedModifierComponent.cs

```diff
@@ -18,9 +18,11 @@ public sealed partial class MovementSpeedModifierComponent : Component
         public const float DefaultWeightlessModifier = 0.7f;
         public const float DefaultWeightlessAcceleration = 1f;
 
-        public const float DefaultAcceleration = 20f;
-        public const float DefaultFriction = 20f;
-        public const float DefaultFrictionNoInput = 20f;
+        // Parkstation-AcceleratingMovement Start
+        public const float DefaultAcceleration = 8f;
+        public const float DefaultFriction = 8f;
+        public const float DefaultFrictionNoInput = 14f;
+        // Parkstation-AcceleratingMovement End
```

Comment:
```suggestion
        public const float DefaultAcceleration = 8f; // Parkstation-AcceleratingMovement
        public const float DefaultFriction = 8f; // Parkstation-AcceleratingMovement
        public const float DefaultFrictionNoInput = 14f; // Parkstation-AcceleratingMovement
```
I'd rather see them done like this simply because we're modifying the default values and not adding anything.

Speaking of, there's no way to do this in YAML? :(